dysnix/op-node/values.yaml

```diff
@@ -0,0 +1,162 @@
+image:
+  repository: us-docker.pkg.dev/oplabs-tools-artifacts/images/op-node
+  tag: ""
+  pullPolicy: Always
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+updateStrategy:
+  type: RollingUpdate
+
+terminationGracePeriodSeconds: 300
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
```

Comment:
WDYT on dropping all capabilities by default ?
It will be good to add following values too:
```yaml
  allowPrivilegeEscalation: false
  # false is the default value, but we specify it explicitly
  privileged: false
```

Reply:
Good point, but needs some tests before it can be added.